### PR TITLE
Update storage refs - top 50

### PR DESCRIPTION
**Pull request by bbenz**
Complete with latest commits

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@bbenz__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bbenz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: _help.pyperf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<ProfilingTarget xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <StandaloneTarget>
+    <PythonInterpreter>
+      <Id>9a7a9026-48c1-4688-9d5d-e5699d47d074</Id>
+      <Version>2.7</Version>
+    </PythonInterpreter>
+    <WorkingDirectory>C:\Users\bbenz\Documents\GitHub\azure-cli</WorkingDirectory>
```
Comment (troydai) [inline _help.pyperf]: The user specific absolute path shouldn't be checked in.

File: _help.pyperf [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <Script>C:\Users\bbenz\Documents\GitHub\azure-cli\src\command_modules\azure-cli-storage\azure\cli\command_modules\storage\_help.py</Script>
+    <Arguments />
+  </StandaloneTarget>
+  <Reports>
+    <Report>
+      <Filename>C:\Users\bbenz\AppData\Local\Temp\_help_20170224.vsp</Filename>
+    </Report>
+  </Reports>
+</ProfilingTarget>
```
Comment (derekbekoe) [inline _help.pyperf]: This file shouldn't be included in the PR.

File: azure-cli.pyproj [hunk 3]
```diff
@@ -422,7 +422,7 @@
     <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\_format.py">
       <SubType>Code</SubType>
     </Compile>
-    <Compile Include="command_modules\azure-cli-storage\azure\cli\command_modules\storage\_help.py">
+    <Compile Include="..\..\..\CLI\Documentation\_help.py">
```
Comment (derekbekoe) [inline azure-cli.pyproj]: Why the change from the original?
Not sure if it's needed.

**Comment by tjprescott** [comment]
@bbenz could you address the comments so we can get this PR merged?

**Comment by troydai** [comment]
Hi @bbenz, I contacted you about this PR last week. Since there isn't any activity since then, we will close this pull request. The closing of the pull request won't impact your branch. Please feel free to open a new pull request whenever you have time to address the comments.

Thanks.
